src/NLog.Extensions.Logging/Logging/NLogMessageParameterList.cs

```diff
@@ -14,23 +14,39 @@ internal class NLogMessageParameterList : IList<NLog.MessageTemplates.MessageTem
         public object OriginalMessage => _originalMessageIndex.HasValue ? _parameterList[_originalMessageIndex.Value].Value : null;
         public int? _originalMessageIndex;
 
+        public bool CustomCaptureTypes => _customCaptureTypes;
+        public bool _customCaptureTypes;
+
+        public bool IsPositional => _isPositional;
+        public bool _isPositional;
```

Comment:
why are those fields public (well internal)? (2 times)

Reply:
internal class NLogMessageParameterList

Reply:
yes, but we have already the public getter. Looks like one of them isn't needed to be public or even there